kratos/modified_shape_functions/modified_shape_functions.h

```diff
@@ -288,7 +288,7 @@ class KRATOS_API(KRATOS_CORE) ModifiedShapeFunctions
     ///@{
 
     const GeometryPointerType mpInputGeometry;
-    const Vector& mrNodalDistances;
+    const Vector mNodalDistances;
```

Comment:
@RiccardoRossi, to avoid a memory leak when running a real case, I needed to store the nodal distances as a copy. This doesn't happen in the modified shape function tests in the core... Any idea?

Reply:
a question: wouldn't it be enough to have a const reference to the distances?

Reply:
It did not work... I don't now the reason...